service/src/test/java/bio/terra/workspace/app/controller/ReferencedGcpResourceControllerGcsBucketConnectedTest.java

```diff
@@ -92,6 +95,20 @@ public void cleanup() throws Exception {
     mockMvcUtils.deleteWorkspace(userAccessUtils.defaultUserAuthRequest(), workspaceId2);
   }
 
+  @AfterEach
+  public void reset() throws Exception {
+    if (bucketIsUpdated) {
+      mockMvcUtils.updateReferencedGcsBucket(
+          userAccessUtils.defaultUserAuthRequest(),
+          workspaceId,
+          sourceResource.getMetadata().getResourceId(),
+          sourceResourceName,
+          RESOURCE_DESCRIPTION,
+          sourceBucketName,
+          ApiCloningInstructionsEnum.NOTHING);
+    }
```

Comment:
I think this should restore `bucketIsUpdated` to `false`, yes?

Reply:
good catch